lib/gshadow.c

```diff
@@ -129,57 +129,37 @@ void endsgent (void)
 	return &sgroup;
 }
 
+
 /*
  * fgetsgent - convert next line in stream to (struct sgrp)
  *
  * fgetsgent() reads the next line from the provided stream and
  * converts it to a (struct sgrp).  NULL is returned on EOF.
  */
 
-/*@observer@*//*@null@*/struct sgrp *fgetsgent (/*@null@*/FILE * fp)
+/*@observer@*//*@null@*/struct sgrp *
+fgetsgent(/*@null@*/FILE *fp)
```

Comment:
Is this null stuff really needed? Can't we use the attribute for it?

Reply:
I added some patches for reducing those comments some time ago, and replaced some of them by attributes and qualifiers

I should have another look at those at some point, and send another round of patches.